src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf),
+    url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+body,
+html,
+form,
+div,
+input {
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+form {
+  margin: 20px;
+  padding: 0;
+}
+
+div {
+  padding: 0;
+}
+
+.input1 {
+  width: 100%;
+  margin: 0 px auto;
+  padding-left: 62px;
```

Comment:
You should set font-weight: 800 for inputs and font-weight: 300 for placeholders (use ::placeholder for styling)